projects/centeva-core/src/components/table/table.component.ts

```diff
@@ -304,91 +289,62 @@ export class TableComponent implements OnInit {
     }
   }
 
+  public setTemplate(column: TableColumn): TemplateRef<unknown> {
```

Comment:
You might be able to refactor this into a Lookup object rather than a function. If you made the column Type explicit with like a string literal or something it would be easy. I often try to use lookup objects esp if I want to bind the value in html for the reasons I said above.